### DISCUSSION TITLE
SN of Inverters displayed but no data

### DISCUSSION THREAD

**Discussion by cccmon** [jpcornil-git/HA-sma#10] [General]
Hello I have installed the integration - everything seems to work fine but I only get the Serial Numbers of the Inverters displayed in HA no more Data from the Webbox. 

My Webbox runs on Firmware: 
01.05.08.R

When I run the python script on my local computer it displays the values in the terminal.

**Comment by jpcornil-git** [comment]
When you go to the integrations panel, how many equipments/entities are reported within the "SMA Webbox" card ?
No errors/warnings reported in Home Assistant logs ?
Could you supply the output log of the python script ?

**Reply by cccmon** [reply]
It looks like this on the SMA Webbox card:
<img width="1212" alt="image" src="https://github.com/jpcornil-git/HA-sma/assets/6532875/79b428d7-9c20-4b86-b56b-3c535eb9bd83">

**Comment by cccmon** [comment]
There are 23 Entities in total in the list:
<img width="1218" alt="image" src="https://github.com/jpcornil-git/HA-sma/assets/6532875/3b4a3450-21c5-4850-bc5d-cc00a2f07f57">

**Comment by cccmon** [comment]
how do I get the output log of the python script ?

I checked also the logs and there are no entries with "sma".

**Comment by jpcornil-git** [comment]
It looks like your entities are just not activated (no entry sign)
=> if you click on one of them, a new popup window will open, click on the gear box sign and then on "Activate".

Wrt homeassistant logs, there should be at least a line like this  (click on load or download the full logs and search for "sma"):
`2023-12-11 10:11:30.380 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration sma_webbox which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant`

Wrt python script, I meant the output of the following command:
`python3 custom_components/sma_webbox/sma_webbox.py <webbox ip_address>`

**Comment by cccmon** [comment]
that sovled the isssue - thanks a lot!

**Comment by jpcornil-git** [comment]
Not really an integration issue but glad I could help :-)
